### PR TITLE
[TH2-846]. Migrate to new version of gRPC generator service.

### PR DESCRIPTION
**Pull request by Ilya-Loginov-Exactpro**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ class CustomDist(sdist):
     def run(self):
         copy_tree(f'src/main/proto/{package_name}', package_name)
 
-        copy_tree(f'src/gen/main/python/{package_name}', package_name)
+        copy_tree(f'src/gen/main/services/python/{package_name}', package_name)
```
Comment (OptimumCode) [inline setup.py:75]: Shouldn't we change the path on line 41?

File: build.gradle [hunk 2]
```diff
@@ -64,7 +62,7 @@ dependencies {
 
     implementation "javax.annotation:javax.annotation-api:1.3.2"
 
-    implementation "com.exactpro.th2:grpc-service-generator:2.0.5"
+    implementation "com.exactpro.th2:grpc-service-generator:3.1.1"
```
Comment (OptimumCode) [inline build.gradle]: Please, extract the generator version to a variable

File: README.md [hunk 3]
```diff
@@ -33,15 +33,11 @@ gradle --no-daemon clean build publish artifactoryPublish \
...
-        ```
-    - Run th2 Python service generator:
-        ```
-        java -jar {path_to_jar} -p src/main/proto/{package_name} -w PythonServiceWriter -o src/gen/main/python/{package_name}
-       ```
+1. Generate services by gradle:
+    ```
+       gradle --no-daemon clean generateProto
+    ```
+    You can find generated files by path: `src/gen/main/services/python`
```
Comment (OptimumCode) [inline README.md]:
```suggestion
    You can find the generated files by the following path: `src/gen/main/services/python`
```

File: setup.py (hunk 1 above)
Comment (Ilya-Loginov-Exactpro) [reply setup.py:75]: I add options `pythonPath` to our service generator and all generated python service's file will be in this path. I think, this command copy service's files.

Comment (OptimumCode) [reply setup.py:75]: Yes, but if we look at line 41 we will see a very similar path. I am not sure but it seems to me that we should also change it.
@RaevskiySV Could you please help us with that? Should we correct the path on line 41 as well?

Comment (RaevskiySV) [reply setup.py:75]: Code on line 41 is responsible for path where generated code from proto files (not services) will be stored.
If we store our services under new directory, we should add another line of copying.
Additionally, we should add `__init__.py` file in order to directory become a Python package.

**Pull request merged**